detox/android/detox/src/full/java/com/wix/detox/espresso/DetoxAction.java

```diff
@@ -158,15 +158,13 @@ public static ViewAction scrollToIndex(int index) {
     }
 
     public static ViewAction setDatePickerDate(String dateString, String formatString) throws ParseException {
+        Date date;
         if (formatString.equals("ISO8601")) {
-            if (Build.VERSION.SDK_INT < 26) {
-                throw new AssertionError("To use DatePicker on Android you must use API 26 or above");
-            }
-            ZonedDateTime date = ZonedDateTime.parse(dateString);
-            return PickerActions.setDate(date.getYear(), date.getMonthValue(), date.getDayOfMonth());
-        } 
+            date = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").parse(dateString);
```

Comment:
Does it work with custom time zones? `+10:00` etc?

Reply:
Oh, I see you changed it again . Probably this is not relevant.

Reply:
> I think the android implementation does not support ms either.

Well, it does not throw errors and it passes the tests. Do you think we need any extra action items in such case? 😕 ❓